controllers/hacker.controller.js

```diff
@@ -58,28 +58,19 @@ async function createHacker(req, res) {
 
 /**
  * @async
- * @function updateHacker
+ * @function updatedHacker
  * @param {*} req
  * @param {*} res
  * @return {JSON} Success or error status
  * @description 
  *      Change a hacker's information based on the hacker's mongoID specified in req.params.id.
```

Comment:
This isn't what this function is doing anymore. Also, a nitpick but it would be great to have a function name that's more representative of the fact that it's only sending back an HTTP response now.

Reply:
Good point, I'll change the description. 

In terms of the function name, I was thinking along the lines of "what we do when we have already updated the hacker"